charts/ingress-nginx/README.md

```diff
@@ -159,7 +159,7 @@ controller:
       enabled: true
       annotations:
         # Create internal ELB
-        service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
+        service.beta.kubernetes.io/aws-load-balancer-internal: true
```

Comment:
This needs to be the string `"true"` not the boolean `true`.

Reply:
I have updated the PR to use "true" in the annotations